docs/install.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Install and Uninstall
+---
+# Install and Uninstall
+
+Users can choose between operating systems of MacOS, Windows, and Linux for Chef InSpec.
+
+## Install Chef InSpec
+
+You can download the latest Chef InSpec package relevant to your operating system at [our Downloads Page](https://downloads.chef.io/inspec).
+
```

Comment:
Might be nice to bring back the button for here as well as the clipboard copy links for the commands below.

See https://www.inspec.io/downloads/

Reply:
Understood. I'm not going to be able to hardcode a button in this first iteration, and it probably won't play well with a site migration. I do like those features though!